src/sfizz/Voice.cpp

```diff
@@ -152,21 +151,29 @@ void sfz::Voice::release(int delay) noexcept
     if (state != State::playing)
         return;
 
-    if (egAmplitude.getRemainingDelay() > delay) {
-        switchState(State::cleanMeUp);
-    } else {
-        egAmplitude.startRelease(delay);
+    if (!region->flexAmpEG) {
+        if (egAmplitude.getRemainingDelay() > delay)
+            switchState(State::cleanMeUp);
```

Comment:
We're missing the `startRelease(delay)` here from the previous logic. Did it go elsewhere?

Reply:
Yes because it's now supposed to be called from MM.
see `ADSREnvelopeSource::release`

Reply:
OK I see.